### ISSUE TITLE
Google Webmaster complains of sitemap error

### ISSUE THREAD

**Issue by CoDanny** [getgrav/grav-plugin-sitemap#41]
The error is: 

The XML Sitemap cannot be parsed because it contains one or more unbound namespace prefixes. For example, this error is generated when <xhtml:link> is found in a Sitemap without prior xmlns:xhtml="http://www.w3.org/1999/xhtml".

When the site is multi-language, it adds this lines to each page:

```
    <xhtml:link rel="alternate" hreflang="en" href="https://www.astroprint.com/astroprint-compatible-printers" />
    <xhtml:link rel="alternate" hreflang="es" href="https://www.astroprint.com/es/astroprint-compatible-printers" />
```

These seem to be problem.

**Comment by CoDanny** [comment]
I see that is already fixed with #40

**Comment by flaviocopes** [comment]
Released an update
